### ISSUE TITLE
Document how to join the organization

### ISSUE THREAD

**Issue by sftim** [kubernetes/contributor-site#284]
Within https://www.k8s.dev/docs/guide/ I didn't find any guide on how to join the Kubernetes org.

We should make that very clear.

/kind feature

**Comment by sftim** [comment]
A link to https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.yml&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E would be a good start.

**Comment by CIPHERTron** [comment]
I think we should first add a link to https://github.com/kubernetes/community/blob/master/community-membership.md which describes what are the responsibilities for various types of roles.

views @sftim ?

**Comment by mrbobbytables** [comment]
I agree with  linking to our community membership guides first - tbh I really don't want an influx of people opening issues without actually looking at the requirements first.

**Comment by sftim** [comment]
If folks have the appetite for it, we could add either a flowchart - see https://www.k8s.dev/blog/2021/12/01/improve-your-documentation-with-mermaid.js-diagrams/ for tips - or even something that uses interaction to guide people to find out what steps to take next (eg, talk to corporate legal for the CLA signoff).

**Comment by mrbobbytables** [comment]
I agree but think we should move this convo to the community repo since the source of the contributor guide is there.
cc @pymander
